spec/support/saml_idp_bad_destination_controller.rb.erb

```diff
@@ -0,0 +1,187 @@
+class SamlIdpController < SamlIdp::IdpController
```

Comment:
I think that's a great idea. It makes it a lot clearer. I'll cherry-pick your refactor onto this branch :)